lib/workload/stateless/stacks/icav2-event-translator/translator_service/Makefile

```diff
@@ -0,0 +1,5 @@
+install:
+	@pip install -r deps/requirements-test.txt
+
+test:
+	@python -m unittest tests/test_icav2_event_translator.py
```

Comment:
Could you change target to:

```
test: install
	@python -m unittest tests/test_icav2_event_translator.py
```

After then, follow register yours like others at the root Makefile.

https://github.com/umccr/orcabus/blob/71f4ae6/Makefile#L28-L36

Reply:
Great thanks, already add it to the test-suite.